generic/tests/netperf.py

```diff
@@ -93,6 +93,7 @@ def env_setup(session, ip, user, port, password):
         error.context("Setup env for %s" % ip)
         ssh_cmd(session, "iptables -F", ignore_status=True)
         ssh_cmd(session, "service iptables stop", ignore_status=True)
```

Comment:
You may use service module from virttest, it helps decide which command should be used.
```
from virttest import remote
from virttest.staging import service

        guest_run = remote.RemoteRunner(session=session)
        iptables_service = service.Factory.create_service("iptables", run=guest_run.run)
        iptables_service.stop()
```